docs/env-vars.md

```diff
@@ -26,21 +26,22 @@
 
 
 # Tool settings
-| Environment Variable                   | Description                 | Default Value            |
-|----------------------------------------|-----------------------------|--------------------------|
-| `AEGIS_OSIDB_SERVER_URL`               | OSIDB REST API host         | `https://localhost:8000` |
-| `AEGIS_OSIDB_RETRIEVE_EMBARGOED`       | Enable retrieving embargoed | `false`                  |
-| `AEGIS_USE_TAVILY_TOOL_CONTEXT`        | Use Tavily search api tool  | `false`                  |
-| `TAVILY_API_KEY`                       | Use Tavily search api tool  |                          |
-| `NVD_API_KEY`                          | Use NVD tool (for public)   |                          |
-| `AEGIS_USE_CWE_TOOL_CONTEXT`           | Use mitre CWE tool          | `true`                   |
-| `AEGIS_USE_LINUX_CVE_TOOL_CONTEXT`     | Use linux kernel tool       | `false`                  |
-| `AEGIS_USE_GITHUB_MCP_TOOL_CONTEXT`    | Use github mcp tool         | `true`                   |
-| `GITHUB_PERSONAL_ACCESS_TOKEN`         | Use linux kernel tool       |                          |
-| `AEGIS_USE_WIKIPEDIA_MCP_CONTEXT`      | Use wikipedia mcp tool      | `false`                  |
-| `AEGIS_USE_PYPI_MCP_CONTEXT`           | Use pypi mcp tool           | `false`                  |
-| `AEGIS_USE_MITRE_NVD_MCP_TOOL_CONTEXT` | Use nvd mitre tool          | `false`                  |
-| `AEGIS_USE_CISA_KEV_TOOL_CONTEXT`      | Use cisa kev tool           | `false`                  |
+| Environment Variable                   | Description                           | Default Value            |
+|----------------------------------------|---------------------------------------|--------------------------|
+| `AEGIS_OSIDB_SERVER_URL`               | OSIDB REST API host                   | `https://localhost:8000` |
+| `AEGIS_OSIDB_RETRIEVE_EMBARGOED`       | Enable retrieving embargoed           | `false`                  |
+| `AEGIS_USE_TAVILY_TOOL_CONTEXT`        | Use Tavily search api tool            | `false`                  |
+| `TAVILY_API_KEY`                       | Use Tavily search api tool            |                          |
+| `NVD_API_KEY`                          | Use NVD tool (for public)             |                          |
+| `AEGIS_USE_CWE_TOOL_CONTEXT`           | Use mitre CWE tool                    | `true`                   |
+| `AEGIS_CWE_TOOL_ALLOWED_CWE_IDS`       | Define mitre CWE tool allowed cwe ids | ``                   |
```

Comment:
Let's not realign the columns on each single change:

1. It makes it difficult to follow the real changes.
2. It causes unnecessary merge conflicts across pull requests open at a time.
3. The newly added row ended up misaligned anyway.

Reply:
apologies once again, I Switch between emacs and pycharm and one of them is doing formatting automagically, will investigate to avoid next time.